less/udata/dropdowns.less

```diff
@@ -0,0 +1,22 @@
+.dropdown {
+    // Desktop: display on hover
+    &.dropdown-hover:hover .dropdown-menu {
+        display: block;
+    }
+
+    // Clickable dropdowns
+    &.open .dropdown-menu {
+        display: inline;
+    }
+
+    // Mobile: display inline when menu is open
+    .navbar-collapse.in &.dropdown-hover .dropdown-menu {
```

Comment:
How do we know this is for mobile? Just by reading the CSS selector, it is not obvious.

Reply:
`.navbar-collapse.in` is the bootstrap selector for the expanded menu on mobile only (`.navbar-collapse` is always visible on desktop)
This is not obvious (and I had to search), this is why I put the comment.

Reply:
BTW, why block the merge and request change ?

Reply:
@noirbizarre ok thanks. I was not sure if there was a more obvious way to signpost this. Something more obvious than a comment. I can live with it :-)

Reply:
Some mecanism in bootstrap are a little bit weird to understand :/